src/Agent.Plugins/PipelineCache/TarUtils.cs

```diff
@@ -127,7 +125,8 @@ public static class TarUtils
                 }
                 catch (Exception e)
                 {
-                    ExceptionDispatchInfo.Capture(e).Throw();
+                    // couldn't start the process, so throw a slightly nicer message about required dependencies:
+                    throw new InvalidOperationException($"Failed to start the required dependency '{process.StartInfo.FileName}'.  Please verify the correct version is installed and available on the path.", e);
```

Comment:
Maybe can also consider using StringUtil.Loc next time :)

Reply:
Yep larger work needed to localize pipeline caching.